xcom/Queries.java

```diff
@@ -176,7 +177,7 @@ public Result<Object> onSubmit(Integer option) {
                 case VIEW_INVENTORY:
                 case ACQUIRE_ITEM:
                     transaction(t -> {
-                        String query = "match $campaign isa campaign, has name $name; get $name;";
+                        String query = "match $campaign isa campaign, has name $name; get $name; sort $name asc;";
```

Comment:
If we run this query twice, the order changed, which messed up the indices in the query test.